docs/source/user.rst

```diff
@@ -0,0 +1,254 @@
+need a Django user account in order to access the admin site to
+perform many of the operations listed below.  For more information
+about the admin site, including how to use it, please see `Chapter 6:
+The Django Admin Site
+<http://www.djangobook.com/en/2.0/chapter06.html>`_.
+
+Production
+----------
+
+.. warning:: It is essential to remember that there are many ways to
+	     brew mead, and that this software was written to suit the
+	     particular brewing :doc:`process <brewprocess>` used by
+	     its author.  That being said, other brewers will likely
+	     find this software useful, and :doc:`pull requests <dev>`
+	     are cheerfully encouraged.
+
+This section focuses on the basics of mead production: ingredients,
+recipes, batches, and products.  A discussion of samples is also
+included.  Each of these models is part of the ``meadery`` app.
+
+Ingredients
+~~~~~~~~~~~
+
+Ingredients are the raw components of the mead.  
+
+There are four types of ingredients: 
+
+* sugars (example: wildflower honey)
+* solvents (example: spring water)
+* flavors (example: bulk cinnamon)
+* yeasts (example: Red Star champagne yeast)
+
+Different types of information are stored for each type of ingredient:
+
+* Specific gravity and specific heat (sugars and solvents)
+* Cost per unit (all)
+* "Natural" or not (all but yeast)
+* Appellation (all but yeast)
+
+Cost per unit is recorded per kilogram for sugars, per liter for
+solvents, "other" for flavor, and per packet or vial for yeast.
+"Natural" is defined as "does not contain added color, artificial
+flavors or synthetic substances" for these purposes.  Appellation
+describes the region from which the ingredient is sourced.
+
+Ingredients are created from the ingredient select page and can be
+changed at the ingredient change page.
+
+Recipes
+~~~~~~~
+
+Recipes are collections of ingredients, including the ingredient's
+amount and (in the case of sugars and solvents) temperature as
+appropriate.  The total cost of the batch can be calculated based on
+the ingredients and their amounts.
+
+Information for the ingredients is also used to determine the
+appropriate BJCP category.  Here are some examples:
+
+* A recipe with apple juice as the only solvent will be classified as
+  a cyser.
+
+* A recipe with spices as the only flavor will be classified as a
+  metheglin.
+
+* A recipe with apple juice *and* spices will be classified as an open
+  category mead.
+
+.. note:: These classifications are just recommendations.  If they are
+          incorrect or incomplete, please submit :doc:`bug reports
+          <bugfeature>` as appropriate.
+
+Recipes also have titles and descriptions for organizational purposes
+as well.
+
+Recipes can be created from the recipe select page as well as from
+existing batches by pressing the 'Create recipe from batch' button on
+the batch's change page.
+
+Batches
+~~~~~~~
+
+Batches are actual instantiations of recipes.  Sometimes brew day
```

Comment:
I like "instantiation" because it means a real instance of the object in question.  It's the difference between a recipe for baked bread and a loaf of bread.